Python_based_scripts/Extracting_Spatial_Data_GBR_Features.ipynb

```diff
@@ -0,0 +1,451 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#Initiliasing the figure\n",
+    "fig = plt.figure()\n",
+    "\n",
+    "#Adding a plot to the figure and assigning projection\n",
+    "ax = plt.axes(projection = ccrs.PlateCarree())\n",
+    "\n",
+    "#Plotting all sites in green\n",
+    "sites.plot(ax = ax, alpha = 0.5, color = 'green')\n",
+    "\n",
+    "#Plotting mainland in red\n",
+    "sites[sites.LOC_NAME_S.str.contains('Mainland')].plot(ax = ax, color = 'red', alpha = 0.5)\n",
+    "\n",
+    "#Adding a basemap of land areas\n",
+    "ax.add_feature(cf.LAND)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Saving GBR features as shapefile\n",
+    "We will need to provide a path to the folder where we want to save our shapefile containing the limits of all features within the GBR Marine Protected Area."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sites.to_file('Outputs/GBR_features.shp')"
```

Comment:
when i run this, it comes back with the following error: `No such file or directory`

i'm assuming it's because I don't have an 'outputs' folder in my directory but might be good to either change where it's stored, or put a line about creating that folder

Reply:
Thanks for that. I have corrected this now. I will include it in the next PR later today. I will merge this now so the URL Eduardo highlighted is correct.